utils/rpms/daos.spec

```diff
@@ -562,6 +562,9 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
+* Thu Apr 06 2022 Li Wei <wei.g.li@intel.com> 2.3.100-2
```

Comment:
minor but this may better be "Thu Apr 07 2022" since April 6 was a Wednesday not Thursday, and since the date will match the debian file changes. Indeed in the RPM build for CentOS 7 step for this PR there are some warnings generated
"[2022-04-11T02:10:32.730Z] warning: bogus date in %changelog: Thu Apr 06 2022 Li Wei <wei.g.li@intel.com> 2.3.100-2"

Reply:
Good catch. Fixed in the merge commit since there was a conflict here.